### PR TITLE
Beta to EA for user consent

### PR DESCRIPTION
**Pull request by mystiberry-okta**
## Description:
- Beta > EA for User consent

### Resolves:

[OKTA-159428](https://oktainc.atlassian.net/browse/OKTA-159428)

### CODE REVIEW

**Comment by mystiberry-okta** [comment]
Question: Shouldn't I at least mention the Consent checkbox in these api docs? Maybe in the top section that introduces the set of APIs.

File: _source/_docs/api/resources/oidc.md [hunk 1]
```diff
@@ -65,7 +65,7 @@ of the callback response.
...
-| display               | How to display the authentication and consent UI. Valid values: `page` or `popup`                                                                                                                                                                                                                                                                                                                      | Query      | String   | FALSE    |
+| display               | How to display the authentication and [consent](/docs/api/resources/apps#add-oauth-20-client-application) UI. Valid values: `page` or `popup`                                                                                                                                                                                                                                                                                                                      | Query      | String   | FALSE    |
```
Comment (danielzeller-okta) [inline _source/_docs/api/resources/oidc.md]: we don't actually support `popup` for the display parameter. currently this doesn't actually change behavior- OKTA-84361 will implement this

File: _source/_docs/api/resources/apps.md [hunk 2]
```diff
@@ -1040,9 +1040,11 @@ Adds an OAuth 2.0 client application. This application is only available to the
...
     | `NONE`            | `TRUSTED`                        | `REQUIRED` or `IMPLICIT`    | Not prompted |
     | `NONE`            | `REQUIRED`                       | `REQUIRED`                  | Prompted     |
-    | `NONE`            | `REQUIRED`                       | `IMPLICIT`                  | Not prompted | <!--If you change this, change the table in /oauth2.md too. Add 'LOGIN' to first three rows when supported -->
+    | `NONE`            | `REQUIRED`                       | `IMPLICIT`                  | Not prompted | <!-- If you change this section, change it in authorization-servers.md as well. Add 'LOGIN' to first three rows when supported -->
 
-> {% api_lifecycle beta %} Note: Apps created on `/api/v1/apps` default to `consent_method=TRUSTED`, while those created on `/api/v1/clients` default to `consent_method=REQUIRED`.
+>Notes
+  * Apps created on `/api/v1/apps` default to `consent_method=TRUSTED`, while those created on `/api/v1/clients` default to `consent_method=REQUIRED`.
+  * If you request a scope that requires consent while using the `client_credentials` flow, an error is returned. Because there is no user, no consent can be given.
```
Comment (royalchan-okta) [inline _source/_docs/api/resources/apps.md]: This and the table above are accurate

File: _source/_docs/api/resources/oidc.md (hunk 1 above)
Comment (mystiberry-okta) [reply _source/_docs/api/resources/oidc.md]: I am removing popup

File: _source/_docs/api/resources/apps.md [hunk 3]
```diff
@@ -1045,6 +1045,7 @@ Adds an OAuth 2.0 client application. This application is only available to the
 >Notes
   * Apps created on `/api/v1/apps` default to `consent_method=TRUSTED`, while those created on `/api/v1/clients` default to `consent_method=REQUIRED`.
   * If you request a scope that requires consent while using the `client_credentials` flow, an error is returned. Because there is no user, no consent can be given.
+  * These properties can be configured in the Okta user interface App Wizard, General tab: `tos_uri`, `policy_uri`, `logo_uri`, and `consent_method`.
```
Comment (danielzeller-okta) [inline _source/_docs/api/resources/apps.md]: Not to confuse things a bit more, but these *are* configurable via /apps API (but not the /clients API)

Comment (mystiberry-okta) [reply _source/_docs/api/resources/apps.md]: fix
the link works (tested).

Comment (mystiberry-okta) [reply _source/_docs/api/resources/apps.md]: thumbs up in slack.

File: _source/_docs/api/resources/oidc.md [hunk 4]
```diff
@@ -65,7 +65,7 @@ of the callback response.
...
-| display               | How to display the authentication and consent UI. Valid values: `page` or `popup`                                                                                                                                                                                                                                                                                                                      | Query      | String   | FALSE    |
+| display               | How to display the authentication and [consent](/docs/api/resources/apps#add-oauth-20-client-application) UI. Valid value: `page` for the Okta-hosted consent page                                                                                                                                                                                                                                                                                                                      | Query      | String   | FALSE    |
```
Comment (royalchan-okta) [inline _source/_docs/api/resources/oidc.md]: we do support `popup` for normal OIDC though, correct?

Comment (mystiberry-okta) [reply _source/_docs/api/resources/oidc.md]: Yikes, yes. Let me give that one more try.

Comment (mystiberry-okta) [reply _source/_docs/api/resources/oidc.md]: Here's my fix: 
fixthefix

Comment (royalchan-okta) [reply _source/_docs/api/resources/oidc.md]: So, turns out i'm dumb and we don't yet, sorry!

**Pull request merged**